pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -306,7 +303,7 @@ func ValidateDiskParameters(disk *CloudDisk, params common.DiskParameters) error
 	return nil
 }
 
-func (cloud *CloudProvider) InsertDisk(ctx context.Context, volKey *meta.Key, params common.DiskParameters, capBytes int64, capacityRange *csi.CapacityRange, replicaZones []string, snapshotID string, multiWriter bool) error {
+func (cloud *CloudProvider) InsertDisk(ctx context.Context, project string, volKey *meta.Key, params common.DiskParameters, capBytes int64, capacityRange *csi.CapacityRange, replicaZones []string, snapshotID string, multiWriter bool) error {
```

Comment:
Since disks cannot be used cross-project, why are we plumbing the project around everywhere? Wouldn't it be easier & less toil to use cloud.project for everything except the snapshot api?

Reply:
My understanding was that we didn't want to override the original project provided in the resource's names.

I can revert these changes if they don't make sense to you as they provide no additional functionalities.

Reply:
Ah, good point. Thank you, I've spent too long between reviewing sessions.

/lgtm
/approve